content/ja/handbook/company/infrastructure-standards/realms/sandbox/labels-tags.md

```
@@ -0,0 +1,100 @@
+
+すべてのラベルとタグには `gl_` プレフィックスを使用します。すべてのキーはアンダースコアを使用します（`snake_case`）。値はハイフンを使用する必要があります（スラグ化された値には `alpha-dash`）が、アンダースコアも許可されます。
+
+### グローバルラベル/タグ
+
+各リソースに適用すべき[グローバルラベルとタグの一覧](/handbook/company/infrastructure-standards/labels-tags/)をご参照ください。
+
+### Sandbox レームラベル/タグ
+
+| スラグ/ラベル/タグ/キー                      | 使用方法         | 人間が読める名前               | ドキュメント |
+|-------------------------------------------|--------------------|--------------------------------|-----------------------------------------------------------------|
+| `gl_sandbox_shutdown_after_days`          | 必須           | シャットダウンまでの日数        | [使用ドキュメント](#shutdown-after-days-gl_sandbox_shutdown_after_days)              |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)